docs/sources/reference/commandline/cli.md

```diff
@@ -893,7 +904,6 @@ Creates a new container.
       --cgroup-parent=""         Optional parent cgroup for the container
       --cidfile=""               Write the container ID to the file
       --cpuset-cpus=""           CPUs in which to allow execution (0-3, 0,1)
-      --cpuset-mems=""           Memory nodes (MEMs) in which to allow execution (0-3, 0,1)
```

Comment:
No idea where that came from - I pulled master, did a rebase, resolved the conflict in fileutils.go, and force pushed.